bootstrap/src/main/java/io/jenkins/jenkinsfile/runner/bootstrap/commands/PipelineRunOptions.java

```diff
@@ -70,4 +70,8 @@ public class PipelineRunOptions extends PipelineOptions {
     @CommandLine.Option(names = { "-pc", "--pipeline-configuration" },
             description = "The Pipeline Configuration File when using the Jenkins Templating Engine")
     public File pipelineConfiguration;
+
+    @CommandLine.Option(names = { "-wlrd", "--write-log-retry-duration" },
+            description = "Duration in seconds to retry writing logs to output stream.")
+    public int writeLogRetryDuration = 60;
```

Comment:
In order to keep the same logic as before, we need to decrease this value to 1s.

Reply:
Yes, that's true. This PR should not change the previous behaviour, but add configurability only.

Reply:
Done in [46a57b4](https://github.com/romanisb/jenkinsfile-runner/pull/2/commits/46a57b45ba37e7064e72d5fe0273106ca4e44233).